Passing const char * instead of QByteArray to printf

http://bugs.gentoo.org/show_bug.cgi?id=304755

Patch written by Kacper Kowalik <xarthisius.kk@gmail.com>

--- src/xine/k9xineplayer.cpp
+++ src/xine/k9xineplayer.cpp
@@ -251,7 +251,7 @@
 
     if ((vo_port = xine_open_video_driver(xine,
                                           m_vo.toAscii(), XINE_VISUAL_TYPE_X11, (void *) &vis)) == NULL) {
-        printf("I'm unable to initialize '%s' video driver. Giving up.\n", m_vo.toAscii());
+        printf("I'm unable to initialize '%s' video driver. Giving up.\n", m_vo.toAscii().constData());
         return ;
     }
 
@@ -270,7 +270,7 @@
     running=false;
     m_title="";
     if ((!xine_open(stream, m_mrl.toUtf8())) || (!xine_play(stream, 0, 0))) {
-        printf("Unable to open mrl '%s'\n", m_mrl.toUtf8());
+        printf("Unable to open mrl '%s'\n", m_mrl.toUtf8().constData());
         return ;
     }
     QTimer::singleShot (500, this, SLOT (updatePosition ()));
